packages/scene-composer/src/components/panels/scene-settings/SceneTagSettingsEditor.tsx

```diff
@@ -103,7 +78,7 @@ export const SceneTagSettingsEditor: React.FC = () => {
               value={String(internalScale)}
               type='number'
               onFocus={() => setFocusInput(true)}
-              onBlur={onInputBlur}
+              onBlur={() => setFocusInput(false)}
```

Comment:
You should useCallback, anonymous functions in this case get redefined in every render, creating additional waste for the GC.

Reply:
Tested this suggestion with the snippet below & it crashed the app due to excessive rerenders. Guessing that I misunderstood your suggestion? 

`onBlur={useCallback(() => setFocusInput(false), [focusInput])}`

Reply:
Yeah it needs to be named:

```
const handleBlur = useCallback(() => setFocusInput(false), []);

onBlur={handleBlur}
```

Reply:
And clearly I missed the established pattern that you were referring to... fixing now.